src/components/Navbar.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import {Link,NavLink} from "react-router-dom"
+
+class Navbar extends React.Component{
+    constructor (props){
+        super(props)
+        this.state ={}
+    }
```

Comment:
Here is another used code you should remove from your code base as suggested in App.js